sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3644,7 +3644,10 @@ def _rsync_down(args) -> None:
             try:
                 os.makedirs(local_log_dir, exist_ok=True)
                 runner.rsync(
-                    source=f'{remote_log_dir}/*',
+                    # Require a `/` at the end to make sure the parent dir
+                    # are not created locally. We do not add additional '*' as
+                    # kubernetes's rsync does not work with an ending '*'.
+                    source=f'{remote_log_dir}/',
```

Comment:
Curious -  does the removal of `*` impact how hidden files are handled? On my mac it does not make any difference and this change should be ok, but [this article](https://www.baeldung.com/linux/rsync-and-hidden-files) seems to suggest it does. Any thoughts?

Reply:
It seems no significant difference between `/*` and `/`, and the latter is more robust. The tailing `*` means the shell expands the pattern to include all files and directories in src before rsync runs, while the latter relies on rsync's own logic to sync the content of src.